src/user-management/utils.js

```diff
@@ -0,0 +1,43 @@
+function addClass(element, className) {
+  element.classList.add(className);
+}
+
+function removeClass(element, className) {
+  element.classList.remove(className);
+}
+
```

Comment:
Let's get rid of these one-liner abstractions. They add more overhead, than additional value